Tests/ProjectDescriptionTests/InfoPlistTests.swift

```diff
@@ -5,17 +5,44 @@ import XCTest
 @testable import ProjectDescription
 
 final class InfoPlistTests: XCTestCase {
-    func test_toJSON() throws {
+    func test_toJSON_when_file() throws {
         let subject = InfoPlist.file(path: "path/Info.plist")
 
         let expected =
             """
             {
                "type": "file",
-               "path": "path/Info.plist"
+               "value": "path/Info.plist"
             }
             """
 
         XCTAssertCodableEqualToJson(subject, expected)
     }
+
+    func test_toJSON_when_dictionary() throws {
```

Comment:
minor suggestion (possibly for next time we're in the area) - we can test 

Manifest Type -> Encode -> Data -> Decode -> ManifestType

And then verify the resulting manifest type equals the original one, this may simplify the test and will ensure both the encode and decode paths are covered in one shot! :)

Reply:
This is a good idea. I'll update it as you suggested.